flamedisx/xenon/x1t_sr1.py

```diff
@@ -20,7 +20,6 @@
 DEFAULT_P_DPE = 0.219
 DEFAULT_EXTRACTION_EFFICIENCY = 0.96
 
-DEFAULT_ELECTRON_LIFETIME = 641e3
```

Comment:
We should leave the constant lifetime as an option

Reply:
ok. i've reintroduced the option of having constant elife.